Add look and feel page

dcwalk opened the pull request:
Closes https://github.com/hyphacoop/organizing/issues/77

This needs refinement, but an initial pass.

benhylau commented on look-and-feel.md:
```diff
@@ -0,0 +1,65 @@
+# Look and Feel
+
+### Name and wordmark
```
How come we skipped a level?

dcwalk replied:
format to size

garrying commented on look-and-feel.md:
```diff
@@ -0,0 +1,65 @@
+> **HYPHA** Worker Co-operative  
+> Hypha Worker Co-operative Inc. [our legal name]  
+
+We use the following wordmark in online spaces, services and social media where we represent the organization (e.g., [Twitter](https://twitter.com/hyphacoop) and [LinkedIn](https://ca.linkedin.com/company/hyphacoop)). 
```
`data-proofer-ignore` attr could go on the LinkedIn anchor to get around the proofer error.

dcwalk replied:
gitbook uses a custom parser based on kramdown from what I can tell (most docs disappearing), so I can't just pass this with the {: } syntax. I'm resolving this problem by moving where the content is (not essential here anyway). :wink:

garrying commented on look-and-feel.md:
```diff
@@ -0,0 +1,65 @@
+🍄 `:mushroom:`  
+⚑  
+
+### Assets and Templates
```
```suggestion
### Assets and templates
```

garrying commented on look-and-feel.md:
```diff
@@ -0,0 +1,65 @@
+
+### Colours
+
+<img src="images/hypha-colours.png" alt="Hypha colours" width="90%"/>
```
Could the text on the Lime swatch be translucent black?

garrying commented on look-and-feel.md:
```diff
@@ -0,0 +1,65 @@
+
+<img src="images/hypha-colours.png" alt="Hypha colours" width="90%"/>
+
+Registration Black   
```
I associate "Registration Black" to the CMYK process black. I think X11 is just "Black"

dcwalk replied:
Okay, was just using the names that I was told were associated!

garrying commented on look-and-feel.md:
```diff
@@ -0,0 +1,65 @@
+We tend to use the following emoji and iconography: 
+
+🍄 `:mushroom:`  
+⚑  
```
```suggestion
⚑  `&#9873;`
```

garrying replied:
Maybe include the 🏴 emoji as well?

dcwalk replied:
Sure, if we want to be anarchists about it ;)

garrying commented on look-and-feel.md:
```diff
@@ -0,0 +1,65 @@
+- Wordmark
+- Image bank
+- Letterhead 
+- Report Template
```
```suggestion
- Report template
```

garrying commented on look-and-feel.md:
```diff
@@ -0,0 +1,65 @@
+
+### Typography
+
+Our **heading typeface** is [Work Sans](https://fonts.google.com/specimen/Work+Sans) and our **secondary (body) typeface** is !!!!. For the web we prefer using the [system font stack](https://css-tricks.com/snippets/css/system-font-stack/) sans-serif typeface.
```
Proxima Nova could be listed as a secondary typeface for now if we go by the deck

dcwalk replied:
Just realized this isn't a free font :flushed:

garrying commented on look-and-feel.md:
```diff
@@ -0,0 +1,65 @@
+
+### Typography
+
+Our **heading typeface** is [Work Sans](https://fonts.google.com/specimen/Work+Sans) and our **secondary (body) typeface** is !!!!. For the web we prefer using the [system font stack](https://css-tricks.com/snippets/css/system-font-stack/) sans-serif typeface.
```
```suggestion
Our **heading typeface** is [Work Sans](https://github.com/weiweihuanghuang/Work-Sans) and our **secondary (body) typeface** is !!!!. For the web we prefer using the [system font stack](https://css-tricks.com/snippets/css/system-font-stack/) sans-serif typeface.
```

dcwalk commented:
Okay, addressed feedback! Stripped out the extra bits on typefaces because I'm running up on limitations on adding fonts and having them viewable in gitbook. Chalk another thing up to using a discontinued tool for this :grimacing:

dcwalk commented:
@garrying if I could get another quick review I think maybe we could merge in the first iteration for now?
